pkg/storegateway/limiter_test.go

```diff
@@ -43,7 +43,7 @@ func TestLimiter(t *testing.T) {
 }
 
 func checkErrorStatusCode(t *testing.T, err error) {
-	st, ok := status.FromError(err)
+	st, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, uint32(http.StatusUnprocessableEntity), code)
```